dimension/dimension.go

```diff
@@ -1,27 +1,43 @@
 package dimension
 
 import (
+	"context"
 	"encoding/json"
 	"net/http"
 
-	"errors"
 	"io"
 	"io/ioutil"
 
 	errs "github.com/ONSdigital/dp-dataset-api/apierrors"
 	"github.com/ONSdigital/dp-dataset-api/models"
 	"github.com/ONSdigital/dp-dataset-api/store"
+	"github.com/ONSdigital/go-ns/audit"
+	"github.com/ONSdigital/go-ns/common"
 	"github.com/ONSdigital/go-ns/log"
 	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
 )
 
-//Store provides a backend for dimensions
+// Store provides a backend for dimensions
 type Store struct {
+	Auditor audit.AuditorService
 	store.Storer
 }
 
-//GetNodes list from a specified instance
-func (s *Store) GetNodes(w http.ResponseWriter, r *http.Request) {
+// List of audit actions for dimensions
+const (
+	PostDimensionsAction = "postDimensions"
+	PutNodeIDAction      = "putNodeID"
+)
+
+var notFound = map[error]bool{
```

Comment:
Consider moving this alongside where the errors are defined and reusing it in your other `handleErr` function in the instance package